db/migrations/sqls/20220426160810-init-up.sql

```diff
@@ -0,0 +1,504 @@
+CREATE TABLE IF NOT EXISTS block
```

Comment:
@upalinski 
Let's chat how did you get this file.

Reply:
ok